tensorflow/compiler/xla/service/llvm_ir/llvm_loop.cc

```diff
@@ -276,7 +276,7 @@ std::vector<llvm::Value*> ForLoopNest::EmitOperandArrayLoopNest(
       AddLoopsForShapeOnDimensions(shape, dimensions, name_suffix);
   // Verify every dimension except the 'dimension_to_skip' dimension was set in
   // the index.
-  for (size_t dimension = 0; dimension < multi_index.size(); ++dimension) {
+  for (int64 dimension = 0; dimension < multi_index.size(); ++dimension) {
```

Comment:
I don't think we need to change anything here. Both the loop counter and the limit should be unsigned